src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -218,7 +263,7 @@ public static partial class AdvApi32
         /// </returns>
         [DllImport(api_ms_win_service_management_l2_1_0, SetLastError = true, CharSet = CharSet.Unicode)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static unsafe extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, void* lpInfo);
+        public static extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, IntPtr lpInfo);
```

Comment:
Hahah. Sorry, but I said I didn't touch these files but these functions insertions are mine. So I guess I tried to mirror the API description too much and we ended up knowing that `void*` and `FriendlyFlags.Optional` are incompatible. Maybe there are a way to implement the generated code to contemplate it, but for now I'll take out this flag from the `void*` pointers